src/solidation/__main__.py

```diff
@@ -351,6 +352,56 @@ def to_markdown(self) -> str:
             pr_durations = [(i.merged_at - i.created_at).days for i in merged_prs]
             if len(pr_durations) > 1:
                 s += f"- PR duration quantiles (days): {quantiles(pr_durations)}\n"
+
+        if self.config.members:
+            s += "##### Issues & PRs assigned to each member:\n"
+            for user in sorted(self.config.members):
+                open_qty = sum(
+                    1
+                    for ip in self.open_ip
+                    if any(u.login == user for u in ip.assignees)
+                )
+                open_pr_qty = sum(
+                    1
+                    for pr in self.open_prs
+                    if any(u.login == user for u in pr.assignees)
+                )
+                open_issue_qty = open_qty - open_pr_qty
+                blocked_qty = sum(
+                    1
+                    for ip in self.open_ip
+                    if any(u.login == user for u in ip.assignees)
+                    and any(lbl.name == "blocked" for lbl in ip.labels)
+                )
+                new_assigned_qty = 0
+                new_closed_qty = 0
+                for ip in self.active_issues + self.active_prs:
+                    if any(u.login == user for u in ip.assignees):
+                        if ensure_aware(ip.created_at) >= since:
+                            new_assigned_qty += 1
+                        if (
+                            ip.closed_at is not None
+                            and ensure_aware(ip.closed_at) >= since
+                        ):
+                            new_closed_qty += 1
+                since_query = quote(since.isoformat(timespec="seconds"))
+                s += (
+                    f"- [{user}](https://github.com/{user}):"
+                    f" [{open_qty}](https://github.com/issues?"
+                    f"q=assignee%3A{user}+is%3Aopen)"
+                    f" = [{open_issue_qty} issues](https://github.com/issues?"
+                    f"q=assignee%3A{user}+is%3Aopen+type%3Aissue)"
+                    f"/[{open_pr_qty} PRs](https://github.com/pulls?"
+                    f"q=assignee%3A{user}+is%3Aopen);"
```

Comment:
I think `/pulls` in the url has no effect, and you still need explicit type
```suggestion
                    f"q=assignee%3A{user}+is%3Aopen+is%3Apr);"
```

Reply:
but also I think the most representative number here should include not only the 'assigned' ones but also PRs authored by the member but not re-assigned to someone else. I am not sure if there is an easy query for that though and found no way to `logical OR` different expressions within query. May be you could craft a separate query, then could be `{assigned} issues/({assigned}+{authored}) PRs`

Reply:
What do you mean by "the most representative number"?  Are you saying that the number of PRs — and only that number — should include authored PRs?

Reply:
> What do you mean by "the most representative number"?

more representative on how many "active" PRs ATM - some are assigned but typically we have PRs simply submitted by people without self-assignment, assuming that they would continue working on them.

> Are you saying that the number of PRs — and only that number — should include authored PRs?

not sure I am fully grasping "only that number" but yes, I was talking about extending the number of PRs (not issues).

Reply:
How should PRs assigned to the same user that created them be counted?  What about PRs assigned to both the creator and another user?

Reply:
if a PR assigned to author (regardless if assigned to anyone else in addition) -- count as assigned. 
In principle I do not care to separate explicitly assigned or authored - I just thought that if there is a view URL to associate with each, then we could separate.